src/test/scala/nl/knaw/dans/easy/schema/PackagingSpec.scala

```diff
@@ -0,0 +1,62 @@
+/**
+ * Copyright (C) 2015-2017 DANS - Data Archiving and Networked Services (info@dans.knaw.nl)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package nl.knaw.dans.easy.schema
+
+import org.scalatest.{FlatSpec, Matchers}
+
+import scala.reflect.io.File
```

Comment:
I notice you use `scala.reflect.io.File` here. How is that different from the `java.io.File` we often use?